### DISCUSSION TITLE
Properties And Fields

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#15] [Progress Tracker]
- What is a field?
- Defining fields.
- Public and Private fields.
- Field naming convention.
- Private field naming convention.
- Limitation of fields.
- Writing getters and setters for fields.
- What is a property?
- Using the prop snippet.
- Difference between property and field.
- Why is it recommended to use properties instead of fields?
- Full property and snippet.
- Read-Only property.
- Writing getters and setters for properties.
- Initializing properties and fields from the constructor.
- Property initializer syntax.
- Exposing private fields publically using properties.
- Implementing data validation using properties.

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by ManikuttyBabu** [comment]
completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by kavitha-BS** [comment]
Completed

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by SivapriyaVB** [comment]
Completed
